addons/account/report/account_aged_partner_balance.py

```diff
@@ -64,7 +64,7 @@ def _get_partner_move_lines(self, account_type, date_from, target_move, period_l
             arg_list += (tuple(partner_ids or [0]),)
         arg_list += (date_from, tuple(company_ids))
 
-        query = '''
+        query = """
```

Comment:
Please avoid useless diff; change only what you need to change; it'll keep the history as clean as possible.